docs/Learning-Environment-Examples.md

```diff
@@ -456,3 +456,43 @@ drop down. New pieces are spawned randomly at the top, with a chance of being
       - Recommended Minimum: 1
       - Recommended Maximum: 20
   - Benchmark Mean Reward: Depends on the number of tiles.
+
```

Comment:
Add these along side the environment addition PRs

Reply:
Removed and moved to the environment PR